validation/validation.txt

```diff
@@ -0,0 +1,2 @@
+validation of tHW (anomalous couplings) implementation
```

Comment:
This file and the summary pdf should be removed. (please remove the validation folder)

Reply:
i deleted the respective folder